x-pack/filebeat/module/zeek/connection/config/connection.yml

```diff
@@ -0,0 +1,53 @@
+type: log
+paths:
+{{ range $i, $path := .connection_paths }}
+ - {{$path}}
+{{ end }}
+exclude_files: [".gz$"]
+tags: {{.tags}}
+
+json.keys_under_root: true
+
+processors:
```

Comment:
This brings up an interesting question on where renaming of fields should happen as we can do it in the Beat or during ingest time. We historically played around with both.

I would recommend to do it in the ingest pipeline instead of on the edge. The advantage that this brings is if we have a raw zeek log from an other shipper in Elasticsearch we can use the same ingest pipeline to reprocess the log. Also if a fix needs to happen to the processing, it can happen to 1 ingest pipeline and the shippers do not have to be modified.

Reply:
There are Pros and Cons for either approach. The intent behind the original proposal is to try to reduce the burden on the ingress nodes as much as possible, considering the log volume of one zeek/bro sensor can be huge, and one can have many sensors in a large deployment.

Reply:
My main worry is that if we want to reprocess a log in ES, we don't have the info there (NOTE: This is a feature we don't have yet).

As this will probably land in 7.0 and changing from doing it on the filebeat side to ingest side would probably be a breaking change, it means our decision here will stay for a while.

Reply:
Sounds reasonable to me. But note that currently both Suricata and Zeek modules put the original data under suricata and zeek struct respectively using Filebeat. The ingress pipelines will not work properly if the raw data is shipped from something else other than beats. Should what you mentioned be a requirement, both modules would need to keep the raw data intact and do everything in the ingress pipelines, including adding the prefixes.

Reply:
@alakahakai Good point, missed that part.